zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4512,6 +4512,37 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
       x-eventually-consistent: true
+
+  /process-definitions/statistics/message-subscriptions:
+    post:
+      tags:
```

🔧 We need to define the consistency behavior for all paths.

```suggestion
      x-eventually-consistent: true
      tags:
```